### PR TITLE
examples: update restore CR

### PR DESCRIPTION
**Pull request by hasbro17**
[skip ci]

### CODE REVIEW

File: example/etcd-backup-operator/backup_cr.yaml [hunk 1]
```diff
@@ -1,7 +1,7 @@
 apiVersion: "etcd.database.coreos.com/v1beta2"
 kind: "EtcdBackup"
 metadata:
-  name: example-etcd-cluster
+  name: example-etcd-cluster-backup
```
Comment (fanminshi) [inline example/etcd-backup-operator/backup_cr.yaml]: if we change name here, we also need to change the readme for backup operator.
`kubectl get EtcdBackup example-etcd-cluster -o yaml` => `kubectl get EtcdBackup example-etcd-cluster-backup -o yaml`
https://github.com/coreos/etcd-operator/tree/master/cmd/backup-operator#verify-status

Comment (hasbro17) [reply example/etcd-backup-operator/backup_cr.yaml]: Yeah I had the README change in my other branch. Reverting this now. Will do it separately.

Comment (fanminshi) [reply example/etcd-backup-operator/backup_cr.yaml]: sounds good!

**Comment by hasbro17** [comment]
fixed PTAL.

**Comment by hongchaodeng** [comment]
lgtm

**Pull request merged**